packages/w/wl-clipboard/package.yml

```diff
@@ -0,0 +1,21 @@
+name       : wl-clipboard
+version    : 2.2.1
+release    : 1
+source     :
+    - https://github.com/bugaevc/wl-clipboard/archive/refs/tags/v2.2.1.tar.gz : 6eb8081207fb5581d1d82c4bcd9587205a31a3d47bea3ebeb7f41aa1143783eb
+homepage   : https://github.com/bugaevc/wl-clipboard
+license    : GPL-3.0-or-later # CHECK ME
+component  : system.utils
+summary    : Command-line copy/paste utilities for Wayland 
+description: |
+    This project implements two command-line Wayland clipboard utilities, wl-copy and wl-paste, that let you easily copy data between the clipboard and Unix pipes, sockets, files and so on.
+builddeps  :
+    - pkgconfig(wayland-client)
+    - pkgconfig(wayland-protocols)
+    - pkgconfig(wayland-scanner)
```

Comment:
`wayland-client` and `wayland-scanner` are both provided by the same package, so only one of them needs to be listed.